src/utils/Types.h

```diff
@@ -16,8 +16,8 @@ enum class NebulaKeyType : uint32_t {
     kVertex            = 0x00000001,
     kEdge              = 0x00000002,
     kIndex             = 0x00000003,
-    kUUID              = 0x00000004,
```

Comment:
Just delete kUUID, no need to keep the same with 1.0 since we don't need it.

Reply:
fixed